mbtiles/src/mbtiles.rs

```diff
@@ -849,4 +1010,51 @@ mod tests {
         assert!(matches!(result, Err(MbtError::AggHashMismatch(..))));
         Ok(())
     }
+
+    #[actix_rt::test]
+    async fn stat() -> MbtResult<()> {
+        let (mut conn, mbt) = open("../tests/fixtures/mbtiles/world_cities.mbtiles").await?;
+        let res = mbt.statistics(&mut conn).await?;
+
+        assert_eq!(
```

Comment:
you can simplify all this code -- if the statistics result supports `Serialize` trait, replace all asserts with `assert_toml_snapshot!(res, @"")`  (although I think YAML will be easier to read - but for that you will need to add `"yaml"` to `insta = { workspace = true, features = ["toml"] }` in the `Cargo.toml` file - then you can use `assert_yaml_snapshot!`.  Once you do that, run `just bless-insta-mbtiles`